template/template.gen.go

```diff
@@ -212,6 +212,23 @@ var (
 					return nil, fmt.Errorf("error type checking for field Severity: Evaluated expression type %v want %v", t, istio_mixer_v1_config_descriptor.STRING)
```

Comment:
what is the plan for this file?  is it intended to be generated?  Or should this just be `template.go` at this point?